twake/backend/node/src/services/files/web/controllers/files.ts

```diff
@@ -48,6 +48,7 @@ export class FileController {
     const data = await this.service.download(params.id, context);
     const filename = data.name.replace(/[^a-zA-Z0-9 -_.]/g, "");
 
+    response.header("Access-Control-Allow-Headers", "*");
```

Comment:
can it be configurable ? for security reason

Reply:
I removed it, not needed anymore